templates/documents/general_list.html

```diff
@@ -4,11 +4,14 @@
 {php}$this->assign('GLOBALS', $GLOBALS);{/php}
 
 <link rel="stylesheet" href="{$GLOBALS.css_header}" type="text/css">
+<link rel="stylesheet" href="{php}echo $GLOBALS['assets_static_relative']{/php}/dropzone-4-3-0/dist/dropzone.css">
 
 {literal}
 {/literal}
 
 <script type="text/javascript" src="{$WEBROOT}/library/js/DocumentTreeMenu.js"></script>
+<script type="text/javascript" src="{php}echo $GLOBALS['assets_static_relative']{/php}/dropzone-4-3-0/dist/dropzone.js"></script>
```

Comment:
per @sjpadgett recs, change this to:
``````
<script type="text/javascript" src="{$GLOBALS.assets_static_relative}/dropzone-4-3-0/dist/dropzone.js"></script>